about/policy/sgid/index.md

```diff
@@ -29,41 +46,52 @@ A corresponding record to the [Data Stewardship spreadsheet](https://docs.google
 - A link the the data page on the AGRC website
 - A link to the SGID Open Data page
 
-The ArcGIS Online Item will reside in the corresponding ISO category folder and contain the appropriate tags. If the data is authoritative, it will be marked as `authoritative`. If the data does not receive updates, it will be tagged as `static`. If the data has historical relevance **but is not the most recent data** it will be tagged as `shelved`. Note: the [AGOL Validator](https://github.com/agrc/agol-validator) project will transfer metadata from the Internal SGID database, including summary, description, and use limitations.
+We will create a hosted feature service of the dataset in our ArcGIS Online organization. The ArcGIS Online item for this feature service will reside in the corresponding ISO category folder and contain the appropriate tags. If the dataset is authoritative, it will be marked as `authoritative`. We anticipate some datasets won't receive future updates, such as the boundary lines for adjacent counties in the surrounding states. These datasets will only exist in ArcGIS Online and will be tagged as `static`.
+
+The [Auditor](https://github.com/agrc/auditor) project will regularly evaluate and update the ArcGIS Online item's information (tags, thumbnail, title, etc) based off of information from a separate table and from the SGID. It will also update the ArcGIS Online item's metadata from the SGID's geodatabase.
```

Comment:
```suggestion
The [Auditor](https://github.com/agrc/auditor) project will regularly evaluate and update the ArcGIS Online item's information (tags, thumbnail, title, etc.) based on information contained in a separate table, and from the internal SGID geodatabase. It will also update the ArcGIS Online item's metadata from the SGID geodatabase.
```